.github/workflows/containers.yaml

```diff
@@ -0,0 +1,71 @@
+name: ironic container builds
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "containers/**"
+      - ".github/workflows/containers.yaml"
+  pull_request:
+    paths:
+      - "containers/**"
+      - ".github/workflows/containers.yaml"
+  workflow_dispatch:
+
+jobs:
+  ghcr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: setup docker buildx
+        uses: docker/setup-buildx-action@v3
+      - name: login to ghcr.io
+        if: ${{ github.event_name != 'pull_request' }}
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Ironic image metadata
+        id: ironic-meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/rackerlabs/openstackhelm/ironic
+          tags: |
+            type=raw,value=latest,enable={{is_default_branch}}
+            type=sha,enable={{is_default_branch}}
+            type=raw,value=2023.1-ubuntu_jammy
+          labels: |
+            org.opencontainers.image.title=ironic with ISO utils
+            org.opencontainers.image.base.name=ghcr.io/rackerlabs/openstackhelm/ironic
```

Comment:
So that's not really correct. This should be the container we're calling `FROM` in the `Dockerfile`, no?

Reply:
Good catch, thanks ! fixed in the rebase.